### PR TITLE
Implemented mv-div-Thaa-Latn-1987

### LINKED ISSUE #89: Implement system `mv-div-thaa-latn-1987` (Maldivian (Divehi) Maldivian Government 1987 system
This issue is to implement the transliteration system of mv-div-thaa-latn-1987.

This system is referred in the GeoNames database as `div_Thaa2Latn_GMV_1988`, with the system title 'Maldivian (Divehi) Maldivian Government 1988 system'.

 Tests should rely on the data extracted for the `div_Thaa2Latn_GMV_1988` system in https://github.com/riboseinc/geonames-transliteration-data .

### PR DESCRIPTION
**Pull request by bilashsaha**
Fixes https://github.com/interscript/interscript/issues/89

### CODE REVIEW

File: maps/mv-div-Thaa-Latn-1987.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
+---
+authority_id: mv
+id: 1987
+language: iso-639-2:div
+source_script: Thaa
+destination_script: Latn
+name: Maldivian (Divehi) Maldivian Government 1987 system
+alias:
+  ogc11122:
+    code: div_Thaa2Latn_GMV_1988
```
Comment (bilashsaha) [inline maps/mv-div-Thaa-Latn-1987.yaml]: @ronaldtse  As per `OGC 11-122r1` this system is named `div_Thaa2Latn_GMV_1988 Maldivian (Divehi) Maldivian Government 1988 system`. But we found out as per https://www.eki.ee/wgrs/rom2_dv.htm , The Maldivian government adopted this system at 1987 and its approved by BGN/PCGN at 1988. So the year should be 1987. So should we keep 1988 or 1987 here?

`

Comment (ronaldtse) [reply maps/mv-div-Thaa-Latn-1987.yaml]: We should have two different YAML files. 

* One for the Maldivian Government one, year 1987. This one should not have an `alias`.
* One for BGN/PCGN adoption of the 1987 system that inherits from the 1987 system, noted with year 1988. This is the one that should contain the `alias` for `ogc11122`.

Comment (bilashsaha) [reply maps/mv-div-Thaa-Latn-1987.yaml]: @ronaldtse I have made the recommended changes. But for some reason ` specs on macos-latest ruby 2.5` is not passing  installing gems. Its throwing  gem installation error for `ruby-debug-ide` . I tried to re-run but same result.

**Comment by ronaldtse** [comment]
Somehow the `debase` gem is failing to install on macOS with Ruby 2.5. I've re-run it but failed at the same place. Very strange.

**Comment by ronaldtse** [comment]
@bilashsaha since nothing in the setup has changed, I'm going to merge this. Thanks!

**Pull request merged**